scripts/update/Updater.php

```diff
@@ -22,15 +22,30 @@
 namespace oat\taoOpenId\scripts\update;
 
 
+use oat\tao\model\accessControl\func\AccessRule;
+use oat\tao\model\accessControl\func\AclProxy;
 use oat\tao\scripts\update\OntologyUpdater;
+use oat\taoOpenId\model\RelyingPartyService;
 
 class Updater extends \common_ext_ExtensionUpdater
 {
     public function update($initialVersion)
     {
         if ($this->isVersion('0.0.1')) {
             OntologyUpdater::syncModels();
+            AclProxy::applyRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/generis.rdf#AnonymousRole', ['ext'=>'taoOpenId','mod' => 'Connect', 'act' => 'callback']));
+
             $this->setVersion('0.0.2');
         }
+
+        if ($this->isVersion('0.0.2')) {
+            $this->getServiceManager()->register(RelyingPartyService::SERVICE_ID, new RelyingPartyService([]));
+            $this->setVersion('0.1.0');
+        }
+
+        if ($this->isVersion('0.1.0')) {
```

Comment:
Single feature branches do not require individual update versions, scripts can be regrouped